Make leap_day=True default for PSM3 (deprecation) (#1511)
pvlib/pvlib-python: docs/sphinx/source/whatsnew/v0.9.2.rst

  where passing localized timezones with large UTC offsets could return
  rise/set/transit times for the wrong day in recent versions of ``ephem``
  (:issue:`1449`, :pull:`1448`)

> * :py:func:`pvlib.iotools.get_psm3` now raises a deprecation warning for
>   single-year requests where the `leap_day` parameter is not specified.

Reviewer: ```suggestion
* :py:func:`pvlib.iotools.get_psm3` now raises a deprecation warning if
  the `leap_day` parameter is not specified in a single-year request.
```
